.gitignore

```diff
@@ -1,5 +1,6 @@
 # text files
 data
+docs
```

Comment:
let's leave this in please so the demo works?

Once we have some tests we can remove this